src/serializer/ResidualHeapSerializer.js

```diff
@@ -1202,7 +1212,6 @@ export class ResidualHeapSerializer {
   }
 
   _serializeAbstractValueHelper(val: AbstractValue): BabelNodeExpression {
-    invariant(val.kind !== "sentinel member expression", "invariant established by visitor");
```

Comment:
Why did you remove this invariant?

Reply:
This is a redundant invariant from previous checkin.
In "_serializeAbstractValue()" function, there is already same invariant.